src/ecalc/libraries/libecalc/common/libecalc/input/file_io.py

```diff
@@ -388,7 +388,7 @@ def unpack_zip(file: IO) -> Tuple[List[ValidEcalcFile], List[InvalidEcalcFile]]:
 
 def _validate_headers(headers: List[str]):
     for header in headers:
-        if not re.match(r"^[A-Za-z][A-Za-z0-9_.,\-\s#+:\/]*$", header):
+        if not re.match(r"^[A-Za-z][A-Za-z0-9_.,\-\s#+:\/]*$", header) or re.match(r"^Unnamed: \d+$", header):
```

Comment:
It is possible (and easier to read) if you split these 2 conditons in 2 separate if statements; if ... elif, then the error message will be more specific; e.g. invalid character in header vs header missing...or?